crates/wasi/src/p2/network.rs

```diff
@@ -0,0 +1,44 @@
+use crate::p2::bindings::sockets::network::{ErrorCode, Ipv4Address, Ipv6Address};
```

Comment:
This also should be a rename, rather than delete and creation

Reply:
Unfortunately, that can't be done with the squash merge strategy.